src/gtk/toga_gtk/widgets/internal/buttons/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .refresh import RefreshButton
```

Comment:
Why is this needed? It's an internal hierarchy; we can import directly from the source of definition.

Reply:
`from internal.buttons import RefreshButton` feels cleaner than `from internal.buttons.refresh import RefreshButton`
`internal.buttons` also provides `ParentPosition` and `ScrollButton`.

Reply:
Cleaner by what measure? You're obscuring where `RefreshButton` is actually defined. This sort of optimisation makes sense when it's an external API and you want to provide a clean API surface (e.g., the `toga.*` namespace) - but for an explicitly internal API, it only serves to make the implementation harder to reason about. If the number of import paths is unwieldy, that perhaps indicates that the package hierarchy itself is the problem.

Reply:
Makes sense.